code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -85,10 +85,15 @@
 
 /obj/machinery/portable_atmospherics/canister/Initialize(mapload, datum/gas_mixture/existing_mixture)
 	. = ..()
+
 	if(existing_mixture)
 		air_contents.copy_from(existing_mixture)
 	else
 		create_gas()
+
+	var/random_quality = rand(0, 100) / 100
```

Comment:
Any reason this isn't just `rand()`?

```suggestion
	var/random_quality = rand()
```

Reply:
Just me forgetting some of the details of some aspects of dm after not coding anything for nearly a year